src/main/java/com/redhat/devtools/intellij/lsp4ij/DocumentContentSynchronizer.java

```diff
@@ -33,12 +33,9 @@
 public class DocumentContentSynchronizer implements DocumentListener {
     private final static Logger LOGGER = LoggerFactory.getLogger(DocumentContentSynchronizer.class);
 
-    private final @Nonnull
-    LanguageServerWrapper languageServerWrapper;
-    private final @Nonnull
-    Document document;
-    private final @Nonnull
-    URI fileUri;
+    private final @Nonnull LanguageServerWrapper languageServerWrapper;
+    private final @Nonnull Document document;
```

Comment:
is the field still necessary now?

Reply:
I know it is not an elegant code,but commons use of document listener (add/remove) is to do it for the same document instance. I use this document instance to remove the listener (even if it could be used).